.github/actions/macos_install/action.yml

```diff
@@ -5,8 +5,8 @@ runs:
   steps:
     - name: Install MPI, OpenMP
       run: |
-        brew install gfortran
         brew install open-mpi
         brew install libomp
+        ln -s gfortran-8 gfortran
```

Comment:
I think you're missing the correct location of the `gfortran-8` executable, and you are placing the symbolic link in a directory that is not in `PATH`.

Reply:
Maybe you are confusing this command with the `alias` command?

Reply:
you are right. This is a problem as the tests run from a different folder. I have been trying to modify the `PATH` on the branch `test_mac_authors`. So far no luck. Do you think I should be creating the symbolic link in a different directory?

Reply:
Try this:
```bash
 ln -s /usr/local/gfortran-8 /usr/local/gfortran
```

Reply:
`ln: /usr/local/gfortran: Permission denied`
I also tried:
```
mkdir bin
        ln -s gfortran-8 bin/gfortran
        echo "PATH=$(pwd)/bin:${PATH}" >> $GITHUB_ENV
        echo "MPI_OPTS='--oversubscribe'" >> $GITHUB_ENV
```
The ln call seems to work but I still get `/bin/sh: gfortran: command not found` probably due to the process spawning a new shell (see issue #517 )